dunstrc

```diff
@@ -76,6 +76,10 @@
     # Horizontal padding.
     horizontal_padding = 8
 
+    # Padding between text and icon.
+    text_icon_padding = 0
```

Comment:
This should be commented by default and set to a non-zero value now.

Reply:
Does it make a difference ? Not to mention none of the other disabled by default variables are commented out. I don't think that would maintain consistency

Reply:
Oh I forgot that setting it to zero disables it